### PR TITLE
CTOR-831 : Refresh hardware-storage-lenovo-sseries-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh hardware-storage-lenovo-sseries-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md [hunk 1]
```diff
@@ -2,42 +2,429 @@
...
+| Hardware | HW-Storage-Lenovo-Sseries-Hardware-SNMP-custom | Contrôle le matériel |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **HW-Storage-Lenovo-Sseries-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias          | Modèle de service                                    | Description                                               | Découverte |
+|:---------------|:-----------------------------------------------------|:----------------------------------------------------------|:----------:|
+| Traffic-Global | HW-Storage-Lenovo-Sseries-Traffic-Global-SNMP-custom | Contrôle le traffic réseau de plusieurs interfaces réseau | X          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md:33]:
```suggestion
| Traffic-Global | HW-Storage-Lenovo-Sseries-Traffic-Global-SNMP-custom | Contrôle le trafic réseau de plusieurs interfaces réseau | X          |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md [hunk 2]
```diff
@@ -2,42 +2,429 @@
...
+|:----------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](./applications-protocol-snmp.md) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **HW-Storage-Lenovo-Sseries-SNMP-custom** |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de services
+
+| Nom de la règle                             | Description                                                            |
+|:--------------------------------------------|:-----------------------------------------------------------------------|
+| HW-Storage-Lenovo-Sseries-SNMP-Traffic-Name | Découvre les interfaces réseau et supervise le statut et l'utilisation |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md:56]:
```suggestion
| HW-Storage-Lenovo-Sseries-SNMP-Traffic-Name | Découvre les interfaces réseau et en supervise le statut et l'utilisation |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md [hunk 3]
```diff
@@ -2,42 +2,429 @@
...
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK: All interfaces are ok | '*interface_name*#status'='up';;;;'*interface_name*#interface.traffic.in.bitspersecond'=20b/s;;;;'*interface_name*#interface.traffic.out.bitspersecond'=20b/s;;;;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md:261]: thresholds?

File: pp/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md [hunk 4]
```diff
@@ -2,42 +2,428 @@
...
+| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options). | --verbose         |             |
+
+</TabItem>
+<TabItem value="Traffic-Global" label="Traffic-Global">
 
-Be sure to have with you the following information:
+| Macro          | Description                                                                                                                                                                                                         | Default value     | Mandatory   |
+|:---------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTER         | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored. To filter on interface names, see --name                                                               | .*                |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md:219]: y'a pas de --name dans le tableau

File: pp/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md [hunk 5]
```diff
@@ -2,42 +2,428 @@
...
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: All interfaces are ok | '*interface_name*#status'='up';;;;'*interface_name*#interface.traffic.in.bitspersecond'=20b/s;;;;'*interface_name*#interface.traffic.out.bitspersecond'=20b/s;;;;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md:260]: thresholds?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md (hunk 5 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md:260]:
```suggestion
OK: All interfaces are ok | '*interface_name*#status'='up';;;;'*interface_name*#interface.traffic.in.bitspersecond'=20b/s;80;90;;'*interface_name*#interface.traffic.out.bitspersecond'=20b/s;80;90;;
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md (hunk 4 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md:219]:
```suggestion
| FILTER         | Define the interface filter on name (this filter supports regular expressions.). If empty, all interfaces will be monitored.                                                               | .*                |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md [hunk 6]
```diff
@@ -2,42 +2,429 @@
...
+| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). | --verbose         |             |
+
+</TabItem>
+<TabItem value="Traffic-Global" label="Traffic-Global">
+
+| Macro          | Description                                                                                                                                                                                                         | Valeur par défaut | Obligatoire |
+|:---------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTER         | Define the interface filter on IDs (OID indexes, e.g.: 1,2,...). If empty, all interfaces will be monitored. To filter on interface names, see --name                                                               | .*                |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp.md:218]:
```suggestion
| FILTER         | Define the interface filter on name (this filter supports regular expressions.). If empty, all interfaces will be monitored.                                                               | .*                |             |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**